core/src/components/picker-column-legacy/picker-column.tsx

```diff
@@ -6,13 +6,13 @@ import { getClassMap } from '@utils/theme';
 
 import { getIonMode } from '../../global/ionic-global';
 import type { Gesture, GestureDetail } from '../../interface';
-import type { PickerColumn } from '../picker/picker-interface';
+import type { PickerColumn } from '../picker-legacy/picker-interface';
 
 /**
  * @internal
  */
 @Component({
-  tag: 'ion-picker-column',
+  tag: 'ion-picker-legacy-column',
```

Comment:
~ The naming of `ion-picker-legacy-column` vs. `ion-picker-column-legacy` caught my eye, especially as the folder name is called `picker-column-legacy`. I'd be curious if others flagged this while reviewing. 

I can understand why we landed on `ion-picker-legacy-column` from a selector perspective, but I think we should be consistent in folder naming since to date the selector has mostly matched the folder name (without `ion-`).

Reply:
Oops that was an oversight on my part. Fixed in https://github.com/ionic-team/ionic-framework/pull/28584/commits/41b520bc60dacba83e2247015bc94cf468a938d8.